src/RevPrefetcher.cc

```diff
@@ -165,8 +174,16 @@ bool RevPrefetcher::InstFetch(uint64_t Addr, bool &Fetched, uint32_t &Inst){
 
 void RevPrefetcher::Fill(uint64_t Addr){
 
+  if( Addr == 0x4bd38ull ){
+    std::cout << "inside Fill" << std::endl;
+  }
+
   // determine if the address is 32bit aligned
```

Comment:
remove address check or wrap in #ifdef

Reply:
resolved in https://github.com/tactcomplabs/rev/commit/e9a5df3d3de81136677d389638c82e5ced6b65c9